var/spack/repos/builtin/packages/pflotran/package.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Pflotran(AutotoolsPackage):
+    """PFLOTRAN is an open source, state-of-the-art massively parallel
+       subsurface flow and reactive transport code.
+    """
+
+    homepage = "http://www.pflotran.org"
+
+    version('develop', hg='https://bitbucket.org/pflotran/pflotran-xsdk')
+
+    depends_on('mpi')
+    depends_on('hdf5@1.8.12+mpi+fortran')
+    depends_on('petsc@develop')
```

Comment:
No, we cannot merge packages that depend on or provide only a `@develop` version; because that circumvents Spack's security.  To see what I mean... when someone says `spack install pflotran`, Spack will be unable to checksum the download from either `pflotran` or `petsc`.  If either project has been compromised, Spack would happily install that compromised code on the user's machine.

This can be corrected by creating a Spack-only "release" for both `petsc` and `pflotran`.  Then you will have an install that you know works predictably.  See here on how to do it:
    https://spack.readthedocs.io/en/latest/workflows.html#upstream-bug-fixes

Basically, you want to add "DIY" versions to both `pflotran` and `petsc` that look something like:
```
# This is not a tagged release of pyproj.
# The changes in this "version" fix some bugs, especially with Python3 use.
version('1.9.5.1.1',
     git='https://github.com/jswhit/pyproj.git',
     commit='0be612cc9f972e38b50a90c946a9b353e2ab140f')
```